exporter/collector/googlemanagedprometheus/naming.go

```diff
@@ -25,6 +25,7 @@ import (
 
 // GetMetricName returns the metric name with GMP-specific suffixes. The.
 func (c Config) GetMetricName(baseName string, metric pmetric.Metric) (string, error) {
+	unknown := isUnknown(metric)
```

Comment:
you could pass `isUnknown()` directly to the functions instead of making a new variable

Reply:
refactored to avoid this